yt_dlp/extractor/francetv.py

```diff
@@ -199,13 +202,12 @@ def sign(manifest_url, manifest_id):
             title += ' - %s' % subtitle
         title = title.strip()
 
-        subtitles = {}
-        subtitles_list = [{
-            'url': subformat['url'],
-            'ext': subformat.get('format'),
-        } for subformat in info.get('subtitles', []) if subformat.get('url')]
-        if subtitles_list:
-            subtitles['fr'] = subtitles_list
+        subtitles.setdefault('fr', []).extend(
```

Comment:
When I was testing this, I noticed that the HLS manifest uses the language code `fra`. Which means the same language may appear here with two different codes. Clearly there is some room for improvement here.

Reply:
Agreed, but like I mentioned in another comment, there doesn't seem to be any standard for language code in subtitles. Let us keep using the short code whenever possible (since that seems most common). But standardizing it everywhere is beyond the scope of this PR

Reply:
I’ll leave it as-is for now then. But it’s something to look into in the future.